src/views/Index.elm

```diff
@@ -0,0 +1,20 @@
+module Index exposing (..)
```

Comment:
Can you elaborate on this name?

Reply:
`Index` refers to the homepage (in this case the only page) of the site. Index refers to `www.domain.com/` or `www.domain.com/index`. 

It's not a naming convention of elm, I just thought it was descriptive of where the view would be shown

Reply:
Gotcha, I was wondering if maybe where it is shown would be orthogonal to what it is doing. I don't really like `Index` all that much because it is in the domain of HTTP convention, vs the domain of this application.

Reply:
I agree. It's now `Homepage`